.github/workflows/hacl-packages-create-branch.yml

```diff
@@ -34,14 +34,20 @@ jobs:
           git config --local user.email "hacl-star@mailo.com"
           git checkout -b $branch
           git add src include
-          git commit -m "[CI] update code"
+          git commit --allow-empty -m "[CI] update code"
           git push --force --set-upstream  origin $branch
       - name: comment pr
         uses: thollander/actions-comment-pull-request@v1
         with:
           message: |
-            [CI] Extensive testing of the generated code on hacl-packages:
-            - [build](https://github.com/cryspen/hacl-packages/actions?query=branch%3Ahacl-star-${{ github.head_ref }}+workflow%3Abuild_pull_request)
-            - [ocaml](https://github.com/cryspen/hacl-packages/actions?query=branch%3Ahacl-star-${{ github.head_ref }}+workflow%3Aocaml)
-            - [benchmarks](https://github.com/cryspen/hacl-packages/actions?query=branch%3Ahacl-star-${{ github.head_ref }}+workflow%3Abenchmark_pull_request)
+            [CI] Important!
+            The code in `dist` is tested on [cryspen/hacl-packages](https://github.com/cryspen/hacl-packages).
+            `dist` is not automatically re-generated, be sure to do it manually.
```

Comment:
> `dist` is not automatically re-generated, be sure to do it manually.

Should we say that a fresh dist for a successful build can be downloaded from CI (e.g., https://everest-ci.paris.inria.fr/build/7308)? So, there is no need to generate it locally.